test/integration/backend_version_test.rb

```diff
@@ -97,4 +97,68 @@ def setup
       assert_not_authorized 'application key is missing'
     end
   end
+
+  test 'when backend_version is 2 and we switch to oauth, there is no complaint about missing app_keys when calling auth endpoints' do
```

Comment:
These tests make me wonder if Porta should delete the existing app keys when a service switches to oauth :thinking:

Reply:
What if it switched by accident and it is immediately followed by another switch back to the original?

Reply:
So this ties in with what I said before: OAuth services (OIDC, really) don't need (nor want) app_keys. And they certainly should not store client secrets. :/

Reply:
Yeah, that's the issue.